src/main/resources/blockly/js/mcreator_blocks.js

```diff
@@ -445,6 +445,54 @@ Blockly.defineBlocksWithJsonArray([
         "output": "Direction",
         "colour": "20"
     },
+    {
+        "type": "time_to_string",
+        "message0": "Time to text",
+        "output": "String",
+        "colour": "%{BKY_TEXTS_HUE}"
+    },
+    {
+        "type": "time_day",
+        "message0": "Get the day",
+        "output": "Number",
+        "colour": "%{BKY_MATH_HUE}"
+    },
+    {
+        "type": "time_hours",
+        "message0": "Get the hours",
+        "output": "Number",
+        "colour": "%{BKY_MATH_HUE}"
+    },
+    {
+        "type": "time_month",
+        "message0": "Get the month",

```

Comment:
I would rename these to eg. "Month as a number" or something like this, same for others